edgecast/client/ec_client.go

```diff
@@ -0,0 +1,276 @@
+package client
```

Comment:
Yes actually, that's a good point. It would help to make it less ambiguous. I was also thinking that much of the core code should be moved into the `internal` folder - it is not really meant to be used by the user. We could expose public models for authentication.